AppCenter/AppCenterTests/MSEncrypterTests.m

```diff
@@ -64,6 +64,7 @@ - (void)testKeyIsRestoredFromKeychain {
 - (void)testKeyIsNull {
 
   // If
+  [MSEncrypter deleteKeyWithTag:self.keyTag];
```

Comment:
@annakocheshkova We don't need this because it's already called in `tearDown` method.